lib/database_connection.rb

```diff
@@ -0,0 +1,33 @@
+# file: lib/database_connection.rb
+
+require 'pg'
+
+# This class is a thin "wrapper" around the
+# PG library. We'll use it in our project to interact
+# with the database using SQL.
+
+class DatabaseConnection
+  # This method connects to PostgreSQL using the 
+  # PG gem. We connect to 127.0.0.1, and select
+  # the database name given in argument.
+  def self.connect
+    if ENV['ENV'] == 'test'
+      database_name = 'chitter_test'
+    else
+      database_name = 'chitter'
```

Comment:
since the word chitter may be used alot naming the database chitter_database may mitigate confusion and gives a more in depth name